src/node_support/node_requestor.ts

```diff
@@ -12,10 +12,16 @@
  * limitations under the License.
  */
 
+import * as Buffer from 'buffer';
+import {ServerResponse} from 'http';
+import * as Url from 'url';
+
 import {AppAuthError} from '../errors';
-import * as request from 'request';
+import {log} from '../logger';
 import {Requestor} from '../xhr';
-import { log } from '../logger';
+
+const https = require('follow-redirects').https;
+const http = require('follow-redirects').http;
```

Comment:
Is this being used somewhere ?

Also, maybe we should check if the `settings.url` is an `https` url to begin with ?

Reply:
Good point. Fixed in second commit.